app/assets/stylesheets/application.scss

```diff
@@ -14,3 +14,15 @@
 
 @import "bootstrap-sprockets";
 @import "bootstrap";
+
+.page-header {
+	margin: 90px 0px 30px;
+}
+
+.alert-success {
+	margin: 60px 0px -70px;
+}
+
+.extramargin {
+	padding: 10px 0px 30px;
+}
```

Comment:
Is there a way to convert all the file in the project to be 2 space indentation rather than tab